pyproject.toml

```diff
@@ -1,2 +1,63 @@
+# https://packaging.python.org/en/latest/tutorials/packaging-projects/
+# https://hatch.pypa.io/latest/config/build/
+
+[project]
+name = "tutor-notes"
+description = "A Tutor plugin for student notes"
+authors = [
+  { name = "Edly" },
+  { email = "hello@edly.io"},
+]
+maintainers = [
+  { name = "eduNEXT" },
+  { email = "contact@edunext.co" },
+]
+license = {file = "LICENSE.txt"}
```

Comment:
let's not use file here as it renders the complete license file on pip show. See Notes section on https://github.com/overhangio/tutor/issues/1190